charts/epinio/100.0.0+up0.9.1/templates/default-app-namespace.yaml

```diff
@@ -0,0 +1,22 @@
+---
+# Default namespace
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: workspace
+  annotations:
+    linkerd.io/inject: enabled
+  labels:
+    # Instruct kubed to copy image pull secrets over.
+    kubed-sync: "registry-creds"
+    app.kubernetes.io/component: "epinio-namespace"
+
+---
+apiVersion: v1
+kind: ServiceAccount
+automountServiceAccountToken: true
+metadata:
+  name: workspace
+  namespace: workspace
+imagePullSecrets:
+- name: registry-creds
```

Comment:
For patching the default serviceaccount across all namespaces created by this chart + adding a default NetworkPolicy in those namespace, I recommend modifying slightly the contents of https://github.com/rancher/charts/blob/dev-v2.6/charts/rancher-monitoring/100.1.3%2Bup19.0.3/templates/rancher-monitoring/hardened.yaml instead.

This is also fine though.